source/packages/services/installer/package.json

```diff
@@ -16,18 +16,18 @@
   },
   "dependencies": {
     "@awssolutions/cdf-config-inject": "workspace:^2.5.2",
-    "@aws-sdk/client-cloudformation": "~3.53.0",
-    "@aws-sdk/client-iam": "~3.53.0",
-    "@aws-sdk/client-iot": "~3.53.0",
-    "@aws-sdk/client-kms": "~3.53.0",
-    "@aws-sdk/client-lambda": "~3.53.0",
-    "@aws-sdk/client-ec2": "~3.53.0",
-    "@aws-sdk/client-rds": "~3.53.0",
-    "@aws-sdk/client-pricing": "~3.53.0",
-    "@aws-sdk/client-s3": "~3.53.0",
-    "@aws-sdk/client-sts": "~3.53.0",
-    "@aws-sdk/client-ssm": "~3.53.0",
-    "@aws-sdk/lib-storage": "~3.53.0",
+    "@aws-sdk/client-cloudformation": "~3.575.0",
+    "@aws-sdk/client-iam": "~3.575.0",
+    "@aws-sdk/client-iot": "3.575.0",
+    "@aws-sdk/client-kms": "~3.575.0",
+    "@aws-sdk/client-lambda": "3.575.0",
```

Comment:
Some of these updates lost the ~. Is that intentional?

Reply:
This is done automatically by running `rush upgrade-interactive --make-consistent`